modules/js/index.js

```diff
@@ -0,0 +1,94 @@
+class BookCollection {
+  constructor() {
+    this.books = this.loadBooks();
+  }
+
+  addBook(title, author) {
+    const book = { title, author };
+    this.books.push(book);
+    this.saveBooks();
+  }
+
+  removeBook(title) {
+    this.books = this.books.filter((book) => book.title !== title);
+    this.saveBooks();
+  }
+
+  displayBooks() {
+    const bookList = document.getElementById('book-lists');
+    bookList.innerHTML = '';
+
+    this.books.forEach((book) => {
+      const li = document.createElement('li');
+      li.textContent = `"${book.title}" by ${book.author}`;
+
+      const removeButton = document.createElement('button');
+      removeButton.classList.add('remove-button');
+      removeButton.textContent = 'Remove';
+      removeButton.dataset.title = book.title;
+
+      li.appendChild(removeButton);
+      bookList.appendChild(li);
+    });
+  }
+
+  saveBooks() {
+    localStorage.setItem('books', JSON.stringify(this.books));
+  }
+
+    loadBooks = () => {
+      const booksData = localStorage.getItem('books');
+      return booksData ? JSON.parse(booksData) : [];
+    };
+}
+
+function toggleSection(sectionId) {
+  const section = document.getElementById(sectionId);
+  const sections = document.getElementsByTagName('section');
+
+  for (let i = 0; i < sections.length; i += 1) {
+    sections[i].style.display = 'none';
+  }
+
+  section.style.display = 'block';
+}
```

Comment:
- [ ] - [ ] Consider using arrow functions instead of the `function` keyword to leverage ES6 features in JavaScript. Arrow functions offer concise syntax and prevent potential scoping bugs. Please give it a try to enhance code readability and maintainability.

- [OPTIONAL] Consider importing the existing `toggleSection` function from `./modules/js/toggleSection.js` instead of creating a new one. This promotes code reuse and follows the DRY principle.

Reply:
- [ ] As mentioned by the previous reviewer, kindly change this function to an arrow function to follow the project requirement (see image)
![Screenshot from 2023-07-19 07-46-27](https://github.com/mumbereh/Awesome-Book/assets/91142432/9574c300-84c0-4ded-be7c-e8ccedaa0d0a)


> * [ ]  - [ ] Consider using arrow functions instead of the `function` keyword to leverage ES6 features in JavaScript. Arrow functions offer concise syntax and prevent potential scoping bugs. Please give it a try to enhance code readability and maintainability.